tendrl/commons/__init__.py

```diff
@@ -1,11 +1,9 @@
 import __builtin__
-import importlib
+from importlib import import_module
 import inspect
-import pkgutil
-
-import maps
+from pkgutil import walk_packages
```

Comment:
The code is more readable when we call say "pkgutil.walk_packages" than just "walk_packages". So keep the old import style.

Also, "walk_packages" is called in this file, it seems you missed out removing pkgutil from the actual function call